src/molecules/TrysilPlug.js

```diff
@@ -16,27 +16,9 @@ import { Labels as DefaultLabels } from './Labels';
 import { LazyProgressiveImage } from './LazyProgressiveImage';
 import { Source } from './Source';
 import { Col } from '../atoms/Col';
-import { AbsolutelyCenteredSvgIcon } from '../atoms/SvgIcon';
```

Comment:
These changes will make TrysilPlug a lot faster, but the changes in this file belongs in a separate PR, as it is removing functionality.